plugins/woocommerce/composer.lock

```diff
@@ -4620,7 +4620,7 @@
         },
         {
             "name": "woocommerce/monorepo-plugin",
-            "version": "dev-update/action-scheduler-3.7.4",
+            "version": "dev-update-action-sch-3.8.1",
```

Comment:
I was puzzling over this a little, but I see it is derived from the branch name 😅